.circleci/config.yml

```diff
@@ -31,6 +35,33 @@ jobs:
       - persist_to_workspace:
           root: .
           paths: .
+  pretend-to-publish:
+    executor: browser-executor
+    steps:
+      - attach_workspace:
+          at: ~/repo
+      - run:
+          name: Log the version we _would_ publish 😂
+          command: |
+            ls
+            pwd
+            PACKAGE_VERSION=$(cat ./package.json \
+              | grep version \
+              | head -1 \
+              | awk -F: '{ print $2 }' \
+              | sed 's/[",]//g')
+            if [[ $PACKAGE_VERSION == *"beta"* ]];
+            then
+              IS_BETA="true"
+            else
+              IS_BETA="false"
+            fi
+            if [ $IS_BETA == "true" ];
+            then
+              echo "here is where I publish --tag beta $PACKAGE_VERSION (YES BETA)"
+            else
+              echo "here is where I publish $PACKAGE_VERSION (NON BETA)"
+            fi
```

Comment:
Here is an example of it working for a pretend beta version: https://app.circleci.com/pipelines/github/fullstorydev/fullstory-browser-sdk/446/workflows/6a9cb969-f2f8-47bc-af7c-8733fa2b9555/jobs/486?invite=true#step-102-27

Here is one of it working with a non-beta version: https://app.circleci.com/pipelines/github/fullstorydev/fullstory-browser-sdk/445/workflows/2013e0dd-8d4e-4af3-ae33-ecbb8370d075/jobs/484?invite=true#step-102-27

Reply:
Yeah, I was looking at that little script yesterday. I like it!

Reply:
![excellentsimpsons](https://emoji.slack-edge.com/T02FE3LFK/excellentsimpsons/7c016b9498f3b3ad.gif)

I'll make the update to replace the deploy-sdk logic and then I think this will be ready to merge and release the beta version.

I can work on the updates to the documentation in the meantime. It maybe makes sense to release the updated readme when we fully release v2.0.0?

Reply:
> It maybe makes sense to release the updated readme when we fully release v2.0.0?

Yes, I agree. Wait to update the README once `v2` is out of beta.